### PR TITLE
Feature/aws boto only

### PR DESCRIPTION
**Pull request by robparrott**
This now works, though CLI could use some work.

Will defer until Stack class is created, which should wrap the interface with the library.

### CODE REVIEW

**Comment by agperson** [comment]
I really like the direction of this! :thumbsup: Yay for removing unnecessary dependencies!

File: nepho/providers/aws_provider.py [hunk 1]
```diff
@@ -66,62 +88,145 @@ def deploy(self):
...
+
+    def _load_aws_connection_settings(self):
+        """Helper method to load up the conenction settings from configs, or the AWS file."""
+        
+        access_key = self.config.get("aws_access_key_id")
+        secret_key = self.config.get("aws_secret_access_key")
+        region = self.config.get("aws_region")
+        
+        try:
+            aws_config_file = os.environ['AWS_CONFIG_FILE']
```
Comment (agperson) [inline nepho/providers/aws_provider.py:189]: If AWS_CONFIG_FILE is not set, we should default to the standard location as of recent versions of awscli (~/.aws/config

File: nepho/providers/aws_provider.py [hunk 2]
```diff
@@ -28,33 +44,39 @@ class AWSProvider(nepho.core.provider.AbstractProvider):
     def __init__(self, config, scenario=None):
         nepho.core.provider.AbstractProvider.__init__(self, config, scenario)
 
-        self.clidriver = self.setup_awscli_driver()
+        self.connection = self._setup_boto_connection()
```
Comment (agperson) [inline nepho/providers/aws_provider.py:36]: Is there any danger of naming conflicts using the generic "connection" name with future commands that connect to all the various providers to get a list of stacks?

Comment (robparrott) [inline nepho/providers/aws_provider.py:36]: This is scoped to the AWS class only. Sp I don't think so.

**Comment by tfhartmann** [comment]
:+1:  from everyone here

**Pull request merged**